Bottom: 8f29a48836e07ad9ba4b1855d5ea7fb57ce91e97
Top:    b093f7002842e4d72aa9be55a4a75ec783b62172
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-15 01:46:48 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
